pkg/approver/approver_test.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
If this is new, should probably update the copyright year
```suggestion
Copyright 2020 The Kubernetes Authors.
```